tools/screenshot-test/src/app/firebase.service.ts

```diff
@@ -71,10 +71,7 @@ export class FirebaseService {
             this._readResults(childSnapshot);
             break;
         }
-        counter++;
-        if (counter === snapshot.numChildren()) {
-          return true;
-        }
+        return ++counter === snapshot.numChildren();
```

Comment:
It's basic syntax in Java, C++, C#... and in all of those languages, I see this as potentially confusing.